python/looker/rtl/auth_token.py

```diff
@@ -6,6 +6,7 @@
 
 class AuthToken():
     """Used to instantiate or check expiry of an AccessToken object"""
+
```

Comment:
again, yapf differences, wonder if we're somehow each using different defaults (I don't see a .style.yapf file anywhere so I presume we're both using the defaults but then why the differences?)

Reply:
Interesting. I use `yapf --style=google --in-place`, is that what you are doing?